tex/gregoriotex-spaces.tex

```diff
@@ -1212,9 +1291,6 @@
 % begindifference is the difference between the beginning of the text and the beginning of the notes. Warning : it can be negative.
 \newdimen\gre@dimen@begindifference\relax%
 
-% the width of the clef
-\newdimen\gre@dimen@clefwidth\relax%
```

Comment:
This dimension needs to be removed from Command_index_internal.tex if it's there.